README.md

```diff
@@ -1,8 +1,106 @@
-# sdkms-pgp
+# sq-sdkms
 
 This crate leverages [sdkms-client-rust][sdkms-client-rust] and
-[Sequoia-PGP][Sequoia] to perform OpenPGP operations with keys stored in the
-[Fortanix Self-Defending Key-Management System][SDKMS].
+[Sequoia-PGP][Sequoia] to perform sensitive OpenPGP operations with keys stored
+in the [Fortanix Self-Defending Key-Management System][SDKMS].
+
+## Motivation
+
+Sequoia-PGP defines the [Signer][sequoia::Signer] and
+[Decryptor][sequoia::Decryptor] traits for low-level cryptographic operations
+with secret key material, and abstracts over these traits for PGP operations.
+This crate implements these traits with secrets stored inside Fortanix SDKMS,
+allowing to produce PGP material and decrypt ciphertexts without the need to
+export secrets bits.
+
+## CLI
+
+The convenient command-line interface `sq-sdkms` is provided in the
+spirit of Sequoia's `sq` and `sqv`. It can be used in combination with
+other PGP implementations. For instance, a file can be
+encrypted with `gpg` and decrypted with `sq-sdkms`, or signed with `sq-sdkms`
+and verified with `sqv` (see [Example usage](#example-usage) below).
+
+The following subcommands are supported. Run `sq-sdkms help` for all possible
+commands, flags, and options.
+
+- `sq-sdkms generate-key`: Automatically create secrets with sensible defaults
+  inside SDKMS (signing primary and encryption subkey) and form the PGP key
+- `sq-sdkms certificate`: For a key created with `sq-sdkms`, retrieve the
+  corresponding Transferable Public Key
+- `sq-sdkms sign-detached`: Given a file, create a detached signature with the
+  PGP key
+- `sq-sdkms decrypt`: Decrypt the given PGP message encrypted to this key
+
+PGP operations that do not require secrets (such as verifying a signature) can
+be performed with any other PGP implementation, such as `sqv`.
+
+## Building
+
+First, install Sequoia-PGP ([instructions][instructions-sequoia]) and
+[rust-mbedtls][rust-mbedtls].
+
+Clone and `cd` to this repository, and build with `cargo build
+--release`. This will create the `sq-sdkms` binary in the `target/release`
+directory.
+
```

Comment:
As long as we don't have https://github.com/rust-lang/rust/issues/65262, maybe we should document the minimum required rust version here?

Reply:
The minimum version that works for me is `1.45.2`.

Reply:
Good idea, thanks